gengo/gengo.py

```diff
@@ -416,7 +416,10 @@ def _raiseForMultipleErrorResponse(self, results):
         raise GengoError(' '.join(messages), error_code)
 
     def _raiseForSingleErrorResponse(self, results):
-        raise GengoError(results['err']['msg'], results['err']['code'])
+        message = results['err']['msg'] if 'msg' in results['err'] else \
+                                        "Internal Server Error"
+        code = results['err']['code'] if 'code' in results['err'] else 500
```

Comment:
🤔 why are we relying on the code returned in the response content instead of HTTP code? Isn't it going to be confusing if we don't have a response content and got a 403 HTTP code but instead of propagating that to the user of this library we tell them that we got a 500?